src/System.Text.Json/src/System/Text/Json/Serialization/PooledBufferWriter.cs

```diff
@@ -8,20 +8,20 @@
 namespace System.Text.Json.Serialization
 {
     // Note: this is currently an internal class that will be replaced with a shared version.
```

Comment:
Is this comment still valid? Are we going to have a pooled version?

Reply:
i.e. is it worth it to have a one-off pooled implementation for the serializer or should we switch over?

Reply:
I believe it's fine for internal use. I will update the comment.